tests/container_disk_test.go

```diff
@@ -128,6 +131,66 @@ var _ = Describe("[rfe_id:588][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				}
 			})
 		})
+		Context("[Serial]should obey the disk verification limits in the KubeVirt CR", func() {
+			AfterEach(func() {
```

Comment:
The `AfterEach` is not necessary. The `tests.BeforeTestCleanup` in the `BeforeEach` restores the default configuration.

Reply:
Got it, removed.